tests/network/BUILD.bazel

```diff
@@ -40,6 +40,7 @@ go_library(
         "//tests/containerdisk:go_default_library",
         "//tests/flags:go_default_library",
         "//tests/framework/checks:go_default_library",
+        "//tests/framework/matcher:go_default_library",
```

Comment:
I don't know where to put this comment, since it applies to the commit msg; I don't think `ThisVMI seems like a useful shorthand.` has any usefulness to the reader. I would remove that sentence.

Plus, there's a typo in "... helps the read ..." . It should be `helps the reader`.

But all in all, I agree this getter is useful - I find it very strange though to invoke it via `matcher.ThisVMI`. But that's besides the point.

Reply:
right, looks like the matcher lib can be broken into two parts, 
1. real getter, 
2. wrapping as a matcher, 

but that is out of scope for current PR

Reply:
Let me know if you think that using `ThisVMI` further would increase your discomfort of where this helper sits.

Reply:
I don't mind it as is, but apparently someone does: https://github.com/kubevirt/kubevirt/pull/7311#pullrequestreview-900278109

Ultimately, I think he's right.